# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/silo/ext2-magic.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- silo-fcf8ffb4fba3ea72353c111ab7dc210eaae89223/second/fs/ext2.c.vanilla	2021-12-01 15:53:35.139648521 +0100
+++ silo-fcf8ffb4fba3ea72353c111ab7dc210eaae89223/second/fs/ext2.c	2021-12-01 15:57:03.345659262 +0100
@@ -363,7 +363,7 @@
 	err = io_channel_read_blk(fs->io, 1, -1024, sb);
 
 	if (ext2_to_cpu_16(sb->s_magic) != EXT2_SUPER_MAGIC) {
-		printf("EXT2 superblock magic is wrong\n");
+		/*printf("No ext2 superblock magic found\n"); */
 		goto out_free_super;
 	}
 
